test/compile/axis/parse.test.ts

```diff
@@ -130,6 +130,70 @@ describe('Axis', () => {
       expect(axisComponent['x'][0].implicit.labelColor).toEqual('red');
     });
 
+    it('should include axis property in axis config style', () => {
+      const model = parseUnitModelWithScale({
+        mark: 'point',
+        encoding: {
+          x: {
+            field: 'a',
+            type: 'quantitative'
+          }
+        },
+        config: {
+          axis: {style: 'foo'},
+          style: {
+            foo: {labelColor: 'red'}
+          }
+        }
+      });
+
+      const axisComponent = parseUnitAxes(model);
+      expect(axisComponent['x'].length).toEqual(1);
+      expect(axisComponent['x'][0].implicit.labelColor).toEqual('red');
+    });
+
+    it('should include axis property in orientation and type based axis config style', () => {
+      const model = parseUnitModelWithScale({
+        mark: 'point',
+        encoding: {
+          x: {
+            field: 'a',
+            type: 'quantitative'
+          }
+        },
+        config: {
+          axisXQuantitative: {labelColor: 'red'}
+        }
+      });
+
+      const axisComponent = parseUnitAxes(model);
+      expect(axisComponent['x'].length).toEqual(1);
+      expect(axisComponent['x'][0].implicit.labelColor).toEqual('red');
+    });
+
+    it('should include axis property in axis config over axis config style', () => {
+      const model = parseUnitModelWithScale({
+        mark: 'point',
+        encoding: {
+          x: {
+            field: 'a',
+            type: 'quantitative'
+          }
+        },
+        config: {
+          axis: {style: 'foo'},
+          axisX: {labelColor: 'blue'},
+          style: {
+            foo: {labelColor: 'red'}
+          }
+        }
+      });
+
+      const axisComponent = parseUnitAxes(model);
+      expect(axisComponent['x'].length).toEqual(1);
+      expect(axisComponent['x'][0].implicit.labelColor).toEqual(undefined); // blue would get applied with Vega's logic
```

Comment:
Shouldn't this be `blue`? If not, could you revise the comment ` blue would get applied with Vega's logic`?

Reply:
This is intentionally not "blue" since config.axisX is a Vega property, so we don't need to output redundant "blue" in the output axis.

That's why I put the comment, but perhaps it's not clear yet.